etc/recipes/lsst-product-configs/meta.yaml

```diff
@@ -13,8 +13,8 @@ build:
   string: "0"
 
 source:
-  git_tag: 1.0.0
-  git_url: https://github.com/mjuric/legacy_configs
+  git_tag: master
+  git_url: https://github.com/jhoblitt/legacy_configs.git
```

Comment:
I'd propose to move this product repo to `lsst-dm` (or `lsst-sqre`?), rename it to lsst-product-config along the way, and change the tag back to an explicit one.

Otherwise you'll inadvertently be building different sources (as the master advances) having the same version number (0.0.1).